### ISSUE TITLE
x3270 4.2ga doesn't build on Apple Silicon

### ISSUE THREAD

**Issue by swhobbit** [pmattes/x3270#35]
MacOS Monterey 12.3.1 (21E258)

**Procedure followed**

git fetch ; git fetch --tags
git pull
git checkout master 
./configure --prefix=/usr/local --disable=tcl3270 --disable-windows 
./configure --prefix=/usr/local --disable-tcl3270 --disable-windows 
make clean all

**Actual error**
```
screen.o select.o tls_passwd_gui.o status.o stmenu.o trace_gui.o x3270.o xaa.o xactions.o xkybd.o xtables.o xutil.o fallbacks.o version.o -L/usr/X11/lib  -L/usr/X11/lib -L../../../obj/arm-apple-darwin21.4.0/lib3270 -l3270 -L../../../obj/arm-apple-darwin21.4.0/lib3270i -l3270i -L../../../obj/arm-apple-darwin21.4.0/lib32xx -l32xx -framework CoreFoundation -framework Security -liconv  -L../../../obj/arm-apple-darwin21.4.0/lib3270stubs -l3270stubs -lXaw -lXmu -lXt -lX11 
Undefined symbols for architecture arm64:
  "_qualified_host", referenced from:
      _pr3287_start_now in lib3270i.a(pr3287_session.o)
ld: symbol(s) not found for architecture arm64
```

**script output of build**

[xbuild.log](https://github.com/pmattes/x3270/files/8647925/xbuild.log)

**Comment by pmattes** [comment]
That symbol is no longer part of the codebase. So it looks like the 'make clean' didn't fully work after the git pull.

Can you try:
  rm -rf obj
  make

**Comment by swhobbit** [comment]
> Can you try:
> rm -rf obj

`make clean && make && sudo make install`                                                                                     

So yea, `make clean` **didn't**

**Comment by pmattes** [comment]
Fixed in master and 4.2.
